app/services/employee_importer.rb

```diff
@@ -0,0 +1,74 @@
+require "slack-ruby-client"
+
+class EmployeeImporter
+  def initialize
+    configure_slack
```

Comment:
not opposed to having this logic in two places, but wonder if we should consider putting the Slack config stuff into one place so `EmployeeImporter` and `EmployeeFinder` can share it...